aws/components/bastion/setup.ftl

```diff
@@ -249,7 +249,7 @@
                         "Ports" : [ "ssh" ],
                         "IPAddressGroups" :
                             sshEnabled?then(
-                                (segmentObject.Bastion.IPAddressGroups)!(segmentObject.IPAddressGroups)![],
+                                (solution.IPAddressGroups)!(segmentObject.Bastion.IPAddressGroups)!(segmentObject.IPAddressGroups)![],
```

Comment:
Might be better to merge these rather than replace, otherwise migrations could be quite confusing. The UNIQUE behaviour on the merge is probably best